src/main/openapi/ds-datahandler-openapi_v1.yaml

```diff
@@ -108,19 +108,27 @@ paths:
                 
                
                 
-  /solr/index/full:
+  /solr/index:
```

Comment:
Fitting rename for now. When we introduce iterative updates matching `/oai/import/{full|update}`, we can rename it again.